plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1373,7 +1374,7 @@ public synchronized boolean configureTunnelNetwork(final long networkId,
                 }
             }
             if (!configured) {
-                final Script cmd = new Script(_ovsTunnelPath, _timeout, s_logger);
+                final Script cmd = new Script(_ovsTunnelPath, _timeout.getMillis(), s_logger);
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.